app/enterprise/changelog.md

```diff
@@ -4,6 +4,195 @@ no_search: true
+into the main Kong Gateway (OSS) repository. We are making a gradual transition. On this
+release:
+
+- Moved AWS-Lambda inside the Kong repo
+  [#7464](https://github.com/Kong/kong/pull/7464).
+- Moved ACME inside the Kong repo
+  [#7464](https://github.com/Kong/kong/pull/7464).
+- Moved Prometheus inside the Kong repo
+  [#7666](https://github.com/Kong/kong/pull/7666).
+- Moved Session inside the Kong repo
+  [#7738](https://github.com/Kong/kong/pull/7738).
+- Moved GRPC-web inside the Kong repo
+  [#7782](https://github.com/Kong/kong/pull/7782).
+- Moved Serverless functions inside the Kong repo
+  [#7792](https://github.com/Kong/kong/pull/7792).
+
+### Fixes
+
+#### Enterprise
+- This release includes a fix for an issue with the Vitals InfluxDB timestamp generation when inserting metrics. 
+- Kong Gateway (Enterprise) no longer exports `consumer_reset_secrets`.
+
+#### Core
+- Balancer retries now correctly set the `:authority` pseudo-header on balancer retries.
+  [#7725](https://github.com/Kong/kong/pull/7725).
+- Healthchecks are now stopped while the Balancer is being recreated.
+  [#7549](https://github.com/Kong/kong/pull/7549).
+- Fixed an issue in which a malformed `Accept` header could cause unexpected HTTP 500.
+  [#7757](https://github.com/Kong/kong/pull/7757).
+- Kong no longer removes `Proxy-Authentication` request header and `Proxy-Authenticate` response header.
+  [#7724](https://github.com/Kong/kong/pull/7724).
+- Fixed an issue where Kong would not sort correctly Routes with both regex and prefix paths.
+  [#7695](https://github.com/Kong/kong/pull/7695)
+
+#### Hybrid Mode
+- Ensure data plane config thread is terminated gracefully, preventing a semi-deadlocked state.
+  [#7568](https://github.com/Kong/kong/pull/7568)
+
+##### CLI
+- `kong config parse` no longer crashes when there's a Go plugin server enabled.
+  [#7589](https://github.com/Kong/kong/pull/7589).
+
+##### Configuration
+- Declarative Configuration parser now prints more correct errors when pointing unknown foreign references.
+  [#7756](https://github.com/Kong/kong/pull/7756).
+- YAML anchors in Declarative Configuration are properly processed.
+  [#7748](https://github.com/Kong/kong/pull/7748).
+
+##### Admin API
+- `GET /upstreams/:upstreams/targets/:target` no longer returns 404 when target weight is 0.
+  [#7758](https://github.com/Kong/kong/pull/7758).
+
+##### PDK
+- `kong.response.exit` now uses customized "Content-Length" header when found.
+  [#7828](https://github.com/Kong/kong/pull/7828).
+
+##### Plugins
+- [Vault Auth](/hub/kong-inc/vault-auth) (`vault-auth)
+  This plugin is now available for selection in the Kong Manager UI, previously it was hidden.
+- [OpenID Connect](/hub/kong-inc/openid-connect) (`openid-connect`)
+  Upgrade to v2.0.x to maintain version compatibility with older data planes.
+  - The OpenID Connect plugin can now handle JWT responses from a `userinfo` endpoint.
+  - The plugin now supports JWE Introspection.
```

Comment:
This should be up in the Features section instead of the Fixes section

Reply:
k, will move